components/colorloupe/index.css

```
@@ -1,53 +1,97 @@
-  height: var(--spectrum-colorloupe-height-adjusted);
+  inline-size: var(--mod-colorloupe-width, var(--spectrum-colorloupe-width));
+  block-size: var(--mod-colorloupe-height, var(--spectrum-colorloupe-height));
 
   position: absolute;
-  transform: translate(0, var(--spectrum-colorloupe-animation-distance));
+  transform: translate(0, var(--mod-colorloupe-animation-distance, var(--spectrum-colorloupe-animation-distance)));
   opacity: 0;
   transform-origin: bottom center;
-  bottom: calc(50% + var(--spectrum-colorloupe-offset));
-  left: calc(50% - calc(var(--spectrum-colorloupe-width-adjusted) / 2));
+  inset-block-end: calc(var(--spectrum-colorhandle-size) + var(--mod-colorloupe-offset, var(--spectrum-colorloupe-offset)));
+  inset-inline: calc(40% - calc(var(--mod-colorloupe-width, var(--spectrum-colorloupe-width)) / 2));
   transition: transform 100ms ease-in-out, opacity 125ms ease-in-out;
-
   pointer-events: none;
 
+  filter:drop-shadow(
+    var(--mod-colorloupe-drop-shadow-x, var(--spectrum-colorloupe-drop-shadow-x))
+    var(--mod-colorloupe-drop-shadow-y, var(--spectrum-colorloupe-drop-shadow-y))
+    var(--mod-colorloupe-drop-shadow-blur, var(--spectrum-colorloupe-drop-shadow-blur))
+    var(--mod-colorloupe-drop-shadow-color, var(--spectrum-colorloupe-drop-shadow-color))
+  );
+
   &.is-open {
     transform: translate(0, 0);
     opacity: 1;
   }
 }
 
-.spectrum-ColorLoupe-outer {
-  stroke-width: var(--spectrum-colorloupe-outer-border-size);
+.spectrum-ColorLoupe-inner-border {
+  fill: var(--spectrum-picked-color);
+  stroke: var(--mod-colorloupe-inner-border-color, var(--spectrum-colorloupe-inner-border-color));
+  stroke-width: var(--mod-colorloupe-inner-border-width, var(--spectrum-colorloupe-inner-border-width));
 }
 
-.spectrum-ColorLoupe-express {
-  display: var(--spectrum-colorloupe-express-visibility);
+.spectrum-ColorLoupe-outer-border {
+  fill: none;
+  stroke: var(--highcontrast-colorloupe-outer-border-color, var(--mod-colorloupe-outer-border-color, var(--spectrum-colorloupe-outer-border-color)));
+  stroke-width: calc(var(--mod-colorloupe-outer-border-width, var(--spectrum-colorloupe-outer-border-width)) + var(--mod-colorloupe-inner-border-width ,var(--spectrum-colorloupe-inner-border-width)));
 }
 
 .spectrum-ColorLoupe-spectrum {
   display: var(--spectrum-colorloupe-spectrum-visibility);
 }
+
+.spectrum-ColorLoupe-express {
+  display: var(--spectrum-colorloupe-express-visibility);
+}
+
+/* these methods handle switching clip path and mask used for express mobile(l) and express desktop(m) */
+.spectrum-colorloupe-express-loupe {
+  clip-path: var(--spectrum-colorloupe-express-loupe);
+}
+
+.spectrum-colorloupe-express-m-borders {
+  display: var(--spectrum-colorloupe-express-m-border);
+}
+
+.spectrum-colorloupe-express-l-borders {
+  display: var(--spectrum-colorloupe-express-l-border);
+}
+
+/* hack: make sure we don't complain about this color being undefined */
```

```suggestion
/* hack: make sure we don't complain about this color being undefined */
```

Maybe include a note here about it being a user-defined value that is dynamic and thus used by but not defined by the styles?

---

yeah I like this, got it updated!